lib/logatron/logatron.rb

```diff
@@ -72,5 +74,14 @@ def msg_id
     def msg_id=(id)
       Logatron::Contexts.msg_id = id
     end
+
+    private
+
+    def exception_message(e, additional)
+      info = additional.is_a?(Hash) ? additional.map { |(k, v)| "#{k}=>#{v}"}.join(', ') : additional
+      info_msg = info.nil? || info.empty? ? '' : "; MORE_INFO( #{info} )"
```

Comment:
I would drop the string interpolation here.

info_msg = info.nil? || info.empty? ? 'No additional information provided' : info